renku_theme/login/messages/messages_en.properties

```diff
@@ -3,3 +3,4 @@ loginFormLabel=Log in with your RenkuLab account
 
 # Identity provider
 identity-provider-login-label=Sign up or Log in with
+termsText=By continuing, you agree to the RenkuLab <a href="/help/tos">Terms of Service</a> and acknowledge that the <a href="/help/privacy">Privacy Policy</a> applies to you.
```

Comment:
I think it would make sense to add `target="_blank" rel="noreferrer noopener"` to the links here.

Reply:
> I meant: is it doable to have a configuration bit which makes this container appear or not?

I see. This _might_ be possible, but I think it would be necessary to change the theme to make this modification anyway, at which point it would be just as easy to modify the `login.css` file and set

```
div#renku-login-terms-container {
  display: none;
}
```

I can mention this in the README, if you think this is helpful.

Reply:
I see, no need to change things here 👍.